pkg/sfu/receiver.go

```diff
@@ -10,30 +10,37 @@ import (
 
 	"github.com/gammazero/workerpool"
 	"github.com/pion/rtcp"
-	"github.com/pion/rtp"
 	"github.com/pion/webrtc/v3"
 	"github.com/rs/zerolog/log"
 
 	"github.com/livekit/livekit-server/pkg/sfu/buffer"
 )
 
+// TrackReceiver defines a interface receive media from remote peer
+type TrackReceiver interface {
+	TrackID() string
+	StreamID() string
+	GetBitrateTemporalCumulative() [3][4]uint64
+	ReadSimulcastRTP(buf []byte, layer uint8, sn uint16) (int, error)
```

Comment:
May be ReadRTP is better?

Reply:
Yeah I think `ReadRTP` is better. Let's do one sweep of it when we consolidate the relay stuff also and clean it up.